capplets/common/gtkrc-utils.c

```diff
@@ -176,6 +176,48 @@ void gtkrc_get_details(gchar* filename, GSList** engines, GSList** symbolic_colo
 	g_scanner_destroy (scanner);
 }
 
+static const GScannerConfig gtk_rc_scanner_config =
+{
+  (
+   " \t\r\n"
+   )            /* cset_skip_characters */,
+  (
+   "_"
+   G_CSET_a_2_z
+   G_CSET_A_2_Z
+   )            /* cset_identifier_first */,
+  (
+   G_CSET_DIGITS
+   "-_"
+   G_CSET_a_2_z
+   G_CSET_A_2_Z
+   )            /* cset_identifier_nth */,
+  ( "#\n" )     /* cpair_comment_single */,
+
+  TRUE          /* case_sensitive */,
+
+  TRUE          /* skip_comment_multi */,
+  TRUE          /* skip_comment_single */,
+  TRUE          /* scan_comment_multi */,
+  TRUE          /* scan_identifier */,
+  FALSE         /* scan_identifier_1char */,
+  FALSE         /* scan_identifier_NULL */,
+  TRUE          /* scan_symbols */,
+  TRUE          /* scan_binary */,
+  TRUE          /* scan_octal */,
+  TRUE          /* scan_float */,
+  TRUE          /* scan_hex */,
+  TRUE          /* scan_hex_dollar */,
+  TRUE          /* scan_string_sq */,
+  TRUE          /* scan_string_dq */,
+  TRUE          /* numbers_2_int */,
+  FALSE         /* int_2_float */,
+  FALSE         /* identifier_2_string */,
+  TRUE          /* char_2_token */,
+  TRUE          /* symbol_2_token */,
+  FALSE         /* scope_0_fallback */,
+};
```

Comment:
New build warning.
debian
https://app.travis-ci.com/github/mate-desktop/mate-control-center/jobs/612020554#L4528
fedora
https://app.travis-ci.com/github/mate-desktop/mate-control-center/jobs/612020555#L1869
```
gtkrc-utils.c:219:1: warning: missing field 'store_int64' initializer [-Wmissing-field-initializers]

};

^

1 warning generated.
```
This is a normal build warning. I do not get why it is approved with a build warning which everybody can see on normal build.
And why nobody takes a lot at travis-ci build logs.

Reply:
Ok sorry, build warning isn't visible with normal warn cflags. 
But in travis-ci.

Reply:
done